client/app/billing/autoRenew/actions/update/form/form.controller.js

```diff
@@ -0,0 +1,41 @@
+import _ from 'lodash';
+import { AUTO_TYPE, RENEWAL_TYPES } from './form.constants';
+
+export default class {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
+    this.AUTO_TYPE = AUTO_TYPE;
+  }
+
+  $onInit() {
+    this.PERIODS = _.map(
+      this.service.possibleRenewPeriod,
+      period => ({
+        period,
+        label: this.$translate.instant('billing_autorenew_service_update_service_period_value', { month: period }),
+      }),
+    );
+    this.RENEWAL_TYPES = _.map(
```

Comment:
Looks like a method for a `Service` :)

Reply:
Yes and no :) 
This is related to `oui-select` behaviour